File: sol/solution/8 Kyu/58e43389acfd3e81d5000a88/Solution.py
#
# Solution
#
import math
def circle_circumference(circle):
    return 2 * math.pi * circle.radius





#
# Sample Tests
#
import codewars_test as test
from solution import circle_circumference
from preloaded import Point, Circle

@test.describe("Fixed Tests")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(round(circle_circumference(Circle(Point(10, 10), 30)), 6), 188.495559)
        test.assert_equals(round(circle_circumference(Circle(Point(25, -70), 30)), 6), 188.495559)
        test.assert_equals(round(circle_circumference(Circle(Point(-15, 5), 0)), 6), 0)
        test.assert_equals(round(circle_circumference(Circle(Point(-15, 5), 12.5)), 6), 78.539816)

File: sol/solution/8 Kyu/58e43389acfd3e81d5000a88/Solution.rb
=begin
Solution
=end
def circle_circumference(circle)
    2 * Math::PI * circle.radius
end





=begin
Sample Tests
=end
def dotest(circle, expected)
  it "Circle.new(Point.new(#{circle.center.x}, #{circle.center.y}), #{circle.radius})" do
    expect(circle_circumference(circle)).to be_within(1e-6).of(expected)
  end
end

describe 'Basic tests' do
  dotest(Circle.new(Point.new(10, 10), 30), 188.495559)
  dotest(Circle.new(Point.new(25, -70), 30), 188.495559)
  dotest(Circle.new(Point.new(-15, 5), 0), 0)
  dotest(Circle.new(Point.new(-15, 5), 12.5), 78.539816)
end